src/main/java/bio/terra/janitor/service/cleanup/flight/GoogleBucketCleanupStep.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.janitor.service.cleanup.flight;
+
+import bio.terra.cloudres.common.ClientConfig;
+import bio.terra.cloudres.google.storage.StorageCow;
+import bio.terra.janitor.db.TrackedResource;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import com.google.cloud.storage.StorageOptions;
+
+public class GoogleBucketCleanupStep implements Step {
+  private final StorageCow storageCow;
+
+  public GoogleBucketCleanupStep() {
+    // Janitor only uses CRL Cows to delete resources. Cleanup is not needed.
+    this.storageCow =
+        new StorageCow(
+            ClientConfig.Builder.newBuilder().setClient("terra-crl-janitor").build(),
+            StorageOptions.getDefaultInstance());
+  }
+
+  @Override
+  public StepResult doStep(FlightContext flightContext) {
```

Comment:
our do cleanup steps need to check a number of conditions and respond accordingly.
We can do this in a follow up PR if you like, but leave a TODO at least.
https://docs.google.com/document/d/13mYVJML_fOLsX1gxQxRJgECUNT27dAMKzEJxUEvtQqM/edit#heading=h.p8sy8wh5t9mj

Reply:
done